apps/prairielearn/src/pages/instructorAssessmentUploads/instructorAssessmentUploads.html.ts

```diff
@@ -119,15 +123,38 @@ function AssessmentUploadCard({
                     <td>
                       <p>
                         Upload a CSV file to set the total assessment score for individual students.
-                        <a data-bs-toggle="collapse" href="#uploadAssessmentScoresHelp"
-                          >Show details...</a
-                        >
+                        <a data-bs-toggle="collapse" href="#uploadAssessmentScoresHelp">
+                          Show details...
+                        </a>
                       </p>
                       <div class="collapse" id="uploadAssessmentScoresHelp">
                         ${CsvHelpAssessmentInstanceScores()}
                       </div>
                     </td>
                   </tr>
+                  <tr>
+                    ${config.devMode
```

Comment:
Should this conditional include the `<tr>`?

Reply:
Yep, good call.

Reply:
Done in cd72a8e.